.github/labels.yml

```diff
@@ -0,0 +1,277 @@
+  description: Improvements or additions to documentation
+  name: documentation
+- color: 0075ca # blue
+  description: ""
+  name: API
+- color: 0075ca # blue
+  description: ""
+  name: API/v2
+- color: 0075ca # blue
+  description: ""
+  name: CLI
+- color: 0075ca # blue
+  description: ""
+  name: component:st2auth
+- color: 0075ca # blue
+  description: ""
+  name: component:st2client
+- color: 0075ca # blue
+  description: ""
+  name: component:st2web
+- color: 0075ca # blue
+  description: ""
+  name: migrations
+- color: 0075ca # blue
+  description: ""
+  name: runners
+- color: 0075ca # blue
+  description: ""
+  name: 'service: action runner'
+- color: 0075ca # blue
+  description: ""
+  name: 'service: api'
+- color: 0075ca # blue
+  description: ""
+  name: 'service: auth api'
+- color: 0075ca # blue
+  description: ""
+  name: 'service: rules engine'
+- color: 0075ca # blue
+  description: ""
+  name: 'service: scheduler'
+- color: 0075ca # blue
+  description: ""
+  name: 'service: workflow engine'
+
+# ##### area/python labels ##### #
+- color: 0075ca # blue
+  description: ""
+  name: python3
+- color: 0075ca # blue
+  description: ""
+  name: unicode
+
+# ##### area/pack_resources labels ##### #
+- color: 003459 # blue
+  description: ""
+  name: action aliases
+- color: 003459 # blue
+  description: ""
+  name: chatops
+- color: 003459 # blue
+  description: ""
+  name: inquiries
+- color: 003459 # blue
+  description: ""
+  name: output schema
+- color: 003459 # blue
+  description: ""
+  name: packs
+- color: 003459 # blue
+  description: ""
+  name: policies
+- color: 003459 # blue
+  description: ""
+  name: 'workflows: action chain'
+- color: 003459 # blue
+  description: ""
+  name: 'workflows: orquesta'
+
+# ##### area/external_services labels ##### #
+- color: 8336ff # blue
+  description: ""
+  name: database
+- color: 8336ff # blue
+  description: ""
+  name: mongodb
+- color: 8336ff # blue
+  description: ""
+  name: nginx
+- color: 8336ff # blue
+  description: ""
+  name: rabbitmq
+
+# ##### area/logging_and_performance labels ##### #
+- color: 8336ff # purple
+  description: ""
+  name: logging
+- color: 8336ff # purple
+  description: ""
+  name: metrics
+- color: 8336ff # purple
+  description: ""
+  name: performance
+
+# ##### area/install labels ##### #
+- color: 50209c # purple
+  description: ""
+  name: deployment
+- color: 50209c # purple
+  description: ""
+  name: Docker
+- color: 50209c # purple
+  description: StackStorm in High Availability
+  name: HA
+- color: 50209c # purple
+  description: ""
+  name: ansible
+- color: 50209c # purple
+  description: ""
+  name: K8s
+- color: 50209c # purple
+  description: Support/issues/PRs on a specific OS
+  name: OS support
+- color: 50209c # purple
+  description: ""
+  name: st2-packages
+
+# ##### area/security labels ##### #
+- color: 63112b # red purple
+  description: ""
+  name: RBAC
+- color: 63112b # red purple
+  description: ""
+  name: security
```

Comment:
And I opted for a more red/purple `#63112b` for the `area/security` labels.
![image](https://user-images.githubusercontent.com/1558590/144345770-f7a869b3-7c85-48f4-8603-29dc973ec004.png)